wallet/test/mocks/mock_transaction_gateway.dart

```diff
@@ -22,6 +22,7 @@ import 'package:transaction_signing_gateway/transaction_signing_gateway.dart'
 // ignore_for_file: invalid_use_of_visible_for_testing_member
 // ignore_for_file: prefer_const_constructors
 // ignore_for_file: unnecessary_parenthesis
+// ignore_for_file: camel_case_types
```

Comment:
These are the files where fixes failed, because of the ignite codegen issues?

Reply:
This is as you see a mock file . That mockito packages uses to create auto generate mocks.